lib/catalog/martyrology.ts

```diff
@@ -3776,6 +3885,10 @@ export const Martyrology: { catalog: MartyrologyCatalog } = {
       name: 'Teresa of Portugal',
       titles: [Titles.Religious],
     },
+    the_penitent_thief: {
```

Comment:
No initial definite article in the key names.

```suggestion
    penitent_thief: {
```

Reply:
@emagnier, did you remove this item from `martyrology.ts`? I can’t find it anymore in the latest commit. 🤔